frontend/src/scenes/feature-flags/FeatureFlagSnippets.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
```

Comment:
non-blocking

not clear how this relates to the group select, but maybe I'm missing something?

Reply:
See the commit: https://github.com/PostHog/posthog/pull/7322/commits/d6399dd8f1764de7b8af96f65e1bf8522ae97197

The file was getting too long so I extracted code that didn't need to be there.